src/pages/Admin/Admin.tsx

```diff
@@ -230,19 +256,14 @@ export const Admin = () => {
   if (!settings) return <LoadingSpinner />;
 
   return (
-    <div
-      style={{
-        display: "grid",
-        gridTemplateColumns: "50% 50%",
-        columnGap: "7px",
-        rowGap: "5px",
-        whiteSpace: "nowrap",
-        alignItems: "center",
-      }}
-    >
-      <H1>Harvest Column Name</H1>
-      <H1>Map from field in Deskpro</H1>
-      {fields}
+    <div style={{ marginTop: "8px", height: "600px" }}>
```

Comment:
Ideally, this 600px should be in the manifest, then it should use 100% height here. So if the app change, for example they're on mobile, it should continue to work as expected.

Reply:
I'm afraid, I did as you said, but it doesn't work. The height isn't set to 600px for the iframe.
<img width="1669" alt="Screenshot 2024-11-04 at 14 10 47" src="https://github.com/user-attachments/assets/7c61039c-23ce-4c86-98e6-68e5fa51308b">

Reply:
That's fixed in [this PR](https://github.com/deskpro/deskpro-product/pull/15841) which we need to get merged in.

Reply:
But your fix applies to the app with the type `"type": "app_modal"`. Here, we have `"type": "app_embedded"`